File: consultasMedicasApp/app/src/main/java/com/example/josea/consultasmedicas/pojo/Usuario.java
package com.example.josea.consultasmedicas.pojo;

/**
 * Created by josea on 23/05/2017.
 */

public class Usuario {

    private int id;
    private String seguridadSocial;
    private String password;
    private String firstName;
    private String lastName;

    public Usuario(int id, String seguridadSocial, String password, String firstName, String lastName) {
        this.id = id;
        this.seguridadSocial = seguridadSocial;
        this.password = password;
        this.firstName = firstName;
        this.lastName = lastName;
    }

    public Usuario() {}

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getSeguridadSocial() {
        return seguridadSocial;
    }

    public void setSeguridadSocial(String seguridadSocial) {
        this.seguridadSocial = seguridadSocial;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    @Override
    public String toString() {
        return "Usuario{" +
                "id=" + id +
                ", seguridadSocial='" + seguridadSocial + '\'' +
                ", password='" + password + '\'' +
                ", firstName='" + firstName + '\'' +
                ", lastName='" + lastName + '\'' +
                '}';
    }
}

File: C9_Server_Rest_&_WEB/clases/src/Usuario.php
<?php
/*
    LOS USUARIOS SOLO PODRÁN LOGEARSE A TRAVÉS DE LA WEB PARA VISUALIZAR SUS CITAS,
    SU CLAVE PRIMARIA CORRESPONDE CON SU NUMERO DE LA SEGURIDAD SOCIAL.
*/

/**
 * @Entity @Table(name="usuario")
 **/
class Usuario{
    
    /**
     * @Id
     * @Column(type="integer") @GeneratedValue
     */
    private $id;
    
    /**
     * @Column(type="string", length=10, unique=false, nullable=false)
     */
    private $seguridadSocial;
    
    /**
     * @Column(type="string", length=40, unique=false, nullable=false)
     */
    private $password;
    
    /**
     * @Column(type="string", length=25, unique=false, nullable=false)
     */
    private $firstName;
    
    /**
     * @Column(type="string", length=25, unique=false, nullable=false)
     */
    private $lastName;
    
    //Metodos getter and setter
    
    function getId(){
        return $this->id;
    }
    
    function setId($id){
        $this->id = $id;
    }
    
    function getSeguridadSocial(){
        return $this->seguridadSocial;
    }
    
    function setSeguridadSocial($numero){
        $this->seguridadSocial = $numero;
    }
    
    function getPassword(){
        return $this->password;
    }
    
    function setPassword($pass){
        $this->password = $pass;
    }
    
    function getFirstName(){
        return $this->firstName;
    }
    
    function setFirstName($name){
        $this->firstName = $name;
    }
    
    function getLastName(){
        return $this->lastName;
    }
    
    function setLastName($lastName){
        $this->lastName = $lastName;
    }
    
    public function getArray(){
        return array(
                "id" => $this->id,
                "seguridadSocial" => $this->seguridadSocial,
                "password" => $this->password,
                "firstName" => $this->firstName,
                "lastName" => $this->lastName
            );
    }
    
    function __toString(){
        return 'Usuario: ' . $this->getId() . ' ' . $this->getSeguridadSocial() . ' ' . 
        $this->getPassword() . ' ' . $this->getFirstName() . ' ' . $this->getLastName();
    }
}
?>